server/forms/bats.js

```diff
@@ -149,6 +149,14 @@ exports.fields = assign(exports.fields, {
     type: 'num',
     uniqueHash: true
   },
+  tragus: {
+    type: '+int',
+    uniqueHash: true
+  },
+  upperMolar: {
+    type: '+int',
+    uniqueHash: true
```

Comment:
this will break existing records, so need to be deployed when all monitorings are uploaded

Reply:
excluded from records hash